CBLServerWrapper/CBLServerWrapper/ServerManager.cs

```diff
@@ -57,5 +66,28 @@ private static void MinecraftServer_Exited(object sender, EventArgs e)
         {
             ConsoleWindow.ServerExited();
         }
+        /// <summary>
+        /// Finds the name of the current world
+        /// </summary>
+        /// <param name="serverDir">Folder where the server is located</param>
+        public static void GetWorldName(string serverDir)
```

Comment:
Why does this function not just return a string with the WorldName instead of storing it into a static variable that isn't used anywhere else?